docs/develop/add-in-manifests.md

```diff
@@ -41,7 +41,11 @@ A manifest file enables an Office Add-in to do the following:
 
 ## Hosting requirements
 
-All image URIs, such as those used for [add-in commands](../design/add-in-commands.md), must support caching in production. The server hosting the image shouldn't return a `Cache-Control` header specifying `no-cache`, `no-store`, or similar options in the HTTP response. However, when you're developing the add-in and making changes to image files, the caching can prevent you from seeing your changes, so using `Cache-Control` headers is advisable in development.
+The following must support caching in production.
+- All image URIs, such as those used for [add-in commands](../design/add-in-commands.md).
+- The event-handling file of an [event-based activation](event-based-activation.md) add-in that runs in a JavaScript-only runtime.
```

Comment:
Maybe I'm being pedantic, but for images we're talking about URIs, but of the JS, it's the file. I might rewrite the first line to say "The following URIs most support caching..."